pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -94,6 +97,7 @@ func NewReconciler(
 	nodeName types.NodeName,
 	desiredStateOfWorld cache.DesiredStateOfWorld,
 	actualStateOfWorld cache.ActualStateOfWorld,
+	populatorHasAddedPods func() bool,
```

Comment:
Imo it's cleaner to expose it via the interface.

And actually this made me think, the bool is not threadsafe, we could read and write to the bool at the same time? I need to add a lock I think